Makefile

```diff
@@ -33,4 +33,7 @@ lint:
 test:
 	go test ./... -timeout=5m
 
+acceptance:
+	TF_ACC=1 go test ./... -timeout=5m
```

Comment:
Is  environment variable the only way to pass parameters to the test suite? If so, please, make it little bit more obviously named (e.g. `RUN_ACCEPTANCE_TEST`).

Reply:
Yes, that's the only way. The variable name must be TF_ACC as specified in the terraform acceptance test framework.
https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests

Reply:
OK, I did not know that the name of the variable was mandated by the Terraform test framework :-/.

Please, **make sure that you implement the cleanup** suggested in the last paragraph of [this section](https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests#acceptance-tests) of the document. In order to avoid interference between subsequent and/or concurrent executions of acceptance tests, we must use globally unique names (ideally including time) for all created resources (in this case cloudexports in Kentik) in each execution of acceptance tests. **Any left-over resources with prefix matching the just completed execution must be removed.** This is imporant because again the Kentik flow exporter will act on the configuration and it will waste (paid) cloud resources to do that.

Reply:
[Sweeper](https://github.com/kentik/terraform-provider-kentik-cloudexport/blob/9ef27777940096ad1227c272050ba73a423e30f4/internal/provider/data_source_cloudexport_item_test.go#L27) added. This will run after the acceptance tests.